frontend/src/components/collective/CollectivePendingExpenses.js

```diff
@@ -1,32 +1,98 @@
 import React from 'react';
 import { Link } from 'react-router';
-import ExpenseItem from '../../components/ExpenseItem';
+
+import ExpenseEmptyState from '../ExpenseEmptyState';
+import UnpaidExpenseItem from './UnpaidExpenseItem';
+
+
+const DEFAULT_EXPENSES_TO_SHOW = 3;
 
 export default class CollectivePendingExpenses extends React.Component {
```

Comment:
Should we really have a different component for PendingExpenses? Shouldn't there just be a `CollectiveExpenses.js` component and each expense could have the pending status?